hub/api/tests/test_api.py

```diff
@@ -1523,21 +1523,28 @@ def test_hidden_tensors(local_ds_generator):
 def test_dataset_copy(memory_ds, local_ds, num_workers, progressbar, index):
     ds = memory_ds
     with ds:
-        ds.create_tensor("image")
+        ds.create_tensor("images/image1")
+        ds.create_tensor("images/image2")
         ds.create_tensor("label")
+        ds.create_tensor("nocopy")
         for _ in range(10):
-            ds.image.append(np.random.randint(0, 256, (10, 10, 3)))
+            ds.images.image1.append(np.random.randint(0, 256, (10, 10, 3)))
+            ds.images.image2.append(np.random.randint(0, 256, (10, 10, 3)))
```

Comment:
why do we need two images instead of one?

Reply:
This is to test specifying groups in the tensors arg. See the `_resolve_tensor_list`.